accounts_acc_test.go

```diff
@@ -110,9 +110,17 @@ var _ = Describe("Accounts", func() {
 			if errTP := client.AccountTeamProjects.Create(
 				account.Account.Id,
 				team.Team.Id,
-				AccountTeamProject{ProjectName: projectName, TeamType: projectType}); errTP != nil {
+				AccountTeamProject{ProjectName: projectName, TeamType: "developer"}); errTP != nil {
```

Comment:
can we actually check in tests that we don't have `projectName` field in the request body? Or mb I'm missing something...

Reply:
This is an acceptance test, and we create real resources on the Aiven side when executing it. The idea here is that we test Create and Update methods, so we are associating a project to a team with one type and then updating to another. Exactly what is the customer is doing who has this issue.